runtime/syntax/checkhealth.vim

```diff
@@ -14,8 +14,8 @@ syn keyword DiagnosticError ERROR[:]
 syn keyword DiagnosticWarn WARNING[:]
 syn keyword DiagnosticOk OK[:]
 " Note: hs=e starts higlighting on the title line (instead of the "===" line).
-syn match helpSectionDelim /^======*\n.*$/hs=e
-highlight helpSectionDelim gui=reverse cterm=reverse
-syn match healthHeadingChar "=" conceal cchar= contained containedin=helpSectionDelim
+syn match healthSectionDelim /^======*\n.*$/hs=e
+hi def healthSectionDelim gui=reverse cterm=reverse
```

Comment:
avoid abbreviations in code.
```suggestion
hi default healthSectionDelim gui=reverse cterm=reverse
```

Reply:
Are `syn` and `hi` fine in this case?

Reply:
`syn` and `hi` are very common, not going to deal with all that right now.

let's focus on what we control in this pr